index.html

```diff
@@ -89,6 +90,21 @@
             }
         }
 
+        /* Override bug GitHub button */
+        .mz-bug-container {
+            width: 240px;
+        }
+
+        @media only screen and (max-width: 400px) {
+            .mz-bug-container {
+                width: 144px;
+            }
+        }
+
+        .mz-bug-github-link {
+            display: none;
```

Comment:
The GitHub link on this was always "https://github.com/mapzen/" - a default value, not really the source of what you were looking at.